File: AntiClown.Web.Front/src/Dto/F1Bingo/F1BingoCardDto.ts
import { F1BingoCardProbabilityDto } from "./F1BingoCardProbabilityDto";

export interface F1BingoCardDto {
  id: string;
  season: number;
  description: string;
  explanation: string | null;
  probability: F1BingoCardProbabilityDto;
  totalRepeats: number;
  completedRepeats: number;
  isCompleted: boolean;
}

File: src/EntertainmentApi/AntiClown.Entertainment.Api.Dto/F1Predictions/Bingo/F1BingoCardDto.cs
namespace AntiClown.Entertainment.Api.Dto.F1Predictions.Bingo;

public class F1BingoCardDto
{
    public Guid Id { get; set; }
    public int Season { get; set; }
    public string Description { get; set; }
    public string? Explanation { get; set; }
    public F1BingoCardProbabilityDto Probability { get; set; }
    public int TotalRepeats { get; set; }
    public int CompletedRepeats { get; set; }
    public bool IsCompleted { get; set; }
}